include/dosbox.h

```diff
@@ -79,8 +79,6 @@ void DOSBOX_SetTicksDone(const int64_t ticks_done);
 void DOSBOX_SetTicksScheduled(const int64_t ticks_scheduled);
 
 class Config;
```

Comment:
You don't need this forward `class Config;` declaration either.  It was on there to declare the pointer.

Reply:
Cheers, removed.